docs/kotlin_extensions.md

```diff
@@ -40,8 +40,18 @@ withinSpan(spanName, parentSpan){
 }
 
 ```
+2. Closeable extension methods:
 
-2. Extensions for OkHttp request builder:
+```kotlin
+// Executes the given [block] function on the [Closeable] instance, intercepts any Exception and sends it in 
+// a RUM error event closing the [Closeable] instance afterwards.
```

Comment:
This should be part of the Markdown, not part of the code sample. The `[…]` syntax won't work as it does in IntelliJ/AndroidStudio

Reply:
Also i'd make it clear that it does work exactly as the `use` Kotlin extension and simply notify Datadog RUM of any error.

Reply:
For the docs I followed the same pattern as for the other extensions in there, I think we should keep the same format. I will remove the [...] though